content/v3/pulls.md

```diff
@@ -187,7 +187,7 @@ Note: The response includes a maximum of 250 commits. If you are working with a
 
 Name | Type | Description
 -----|------|-------------
-`commit_message`|`string`| The message that will be used for the merge commit
+`commit_message`|`string`| The message that will be used for the merge commit. Note that the text "Merge pull request #PR from user/branch" will be prepended to the message.
```

Comment:
I think we can keep this concise and more futureproof with something like:

> Extra detail to append to automatic commit message.

@illicitonion what do you think?